src/test/java/org/folio/moduserstest/ExpirationIT.java

```diff
@@ -107,4 +124,10 @@ void cannotTriggerExpirationForUnknownTenant() {
     expirationClient.attemptToTriggerExpiration("made-up-tenant")
       .statusCode(is(HTTP_INTERNAL_ERROR));
   }
+
+  @AfterAll
+  public static void after(VertxTestContext context) {
+    context.completeNow();
```

Comment:
I think context completion should be performed in the end, after all operations finished. Otherwise this could hide a bigger problem. 
Moreover, I didnt find any removal of created tenants at the end of tests. Probably this would be a good verification of all async handlers completed and all database transactions finished

Reply:
Rewritten by moving initialization logic to abstract class